commands/deploy.go

```diff
@@ -99,6 +98,9 @@ func (c *DeployCmd) Run() (err error) {
 	}
 	req.SetBasicAuth(username, password)
 
+	client := &http.Client{
+		Timeout: 30 * time.Second,
```

Comment:
Unrelated to this PR but if the content is too big, 30 seconds might not be enough to deploy it. I wonder if this needs to be configurable with a 30s default.

Reply:
I've been thinking about this as well. 

I think the timeout should be 

- 60s by default
- configurable on the command line

Reply:
Fixed in 0338747.